src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -424,21 +424,38 @@ private EmailWrapper generateFeedbackSessionEmailBaseForStudents(
         return email;
     }
 
+    private String generateInstructorFragment(String courseId, String courseName) {
+
+        String fragmentParameters;
+
+        /* If the course name == "", we understand that this method was called by the
+        generateFeedbackSessionEmailBaseForInstructors, which does not need the course name */
```

Comment:
@leeyimin Ok, I will do it very soon but please do not tell me that I have to revert it after I have done it.